src/studio/src/designer/frontend/ux-editor/containers/Container.tsx

```diff
@@ -443,7 +443,7 @@ export class ContainerComponent extends React.Component<IContainerProps, IContai
             xs={12}
           >
             {!this.props.itemOrder?.length && this.renderContainerPlaceholder()}
-            {this.state.expanded && this.props.itemOrder?.length &&
+            {this.state.expanded && (this.props.itemOrder?.length > 0) &&
```

Comment:
Maybe at some point we should refactor these files, so they are a bit shorter. (Also it would be nice to get the markup and functions separated at some point.

Reply:
Totally agree.